tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -8,6 +8,22 @@ cacl/test_cacl_application.py::test_cacl_application:
       - "asic_type in ['broadcom']"
       - "topo_name in ['dualtor', 'dualtor-56', 'dualtor-120']"
 
+#######################################
+#####            copp             #####
+#######################################
+copp/test_copp.py::TestCOPP::test_add_new_trap:
+  xfail:
+    reason: "'Add always_enabled field to coppmgr' is not merged into 202012 yet, a 'strict' param will remind us to remove this mark condition by setting 'xpass' as 'fail'"
+    strict: True
+    conditions:
+      - "release in ['202012']"
+
+copp/test_copp.py::TestCOPP::test_trap_config_save_after_reboot:
+  skip:
```

Comment:
why is this skip and not xfail?

Reply:
This one raises a error during ending up, which can't be tolerated by xfail.
And both of the two test cases succeeded in physical testbeds with 'master' branch, 
so we can just observe 'test_add_new_trap', if it failed, remove these two marks.